command/plugin_info.go

```diff
@@ -78,22 +78,17 @@ func (c *PluginInfoCommand) Run(args []string) int {
 	var pluginNameRaw, pluginTypeRaw string
 	args = f.Args()
 	switch {
-	case len(args) < 1:
-		c.UI.Error(fmt.Sprintf("Not enough arguments (expected 1 or 2, got %d)", len(args)))
+	case len(args) < 2:
```

Comment:
nit: it seems we are repeatedly calling `len()`, may want to factor out a variable for it. Also I wonder if the switch case could just become `if len(args) != 2 ...` to make it a bit simpler. The error message could be simplified with something like: `Invalid number of arguments %d, required 2`.